Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/oryx/build:latest
+FROM mcr.microsoft.com/oryx/build:github-actions
```

Comment:
let us pin the image tag to a specific version...so instead of `github-actions`, it would be `github-actions-20200229.2`